src/templates/helpers/filter.rs

```diff
@@ -0,0 +1,114 @@
+use handlebars::{
+    Context, Handlebars, Helper, HelperDef, RenderContext, RenderError,
+    ScopedJson,
+};
+use serde_json::{json, Value, Map};
+
+#[derive(Clone, Copy)]
+pub struct FilterHelper;
+
+pub enum FilterableValues {
+    Array(Vec<Value>),
+    Object(Map<String, Value>), // imported from serde_json
```

Comment:
Feel free to remove this comment

```suggestion
    Object(Map<String, Value>),
```